src/store_rebuild.h

```diff
@@ -26,6 +33,8 @@ class StoreRebuildData
     int badflags = 0;       /* # bad e->flags */
     int bad_log_op = 0;
     int zero_object_sz = 0;
+    int validatedCount = 0; ///< the number of validated entries
+    timeval startTime = {0, 0}; ///< when the rebuild has started
```

Comment:
We use an overall-better way to initialize PODs in modern code:

```suggestion
    timeval startTime = {}; ///< when the rebuild has started
```